src/sync/pull-request.ts

```diff
@@ -87,9 +98,19 @@ export default async function(
 	const pullRequests = (
 		await Promise.all(
 			edges.map(async (pull) => {
+				const prDetails = useNewGHClient ? await newGithub.getPullRequest(repository.owner.login, repository.name, pull.number) :
+					(await github?.pulls?.get({
+						owner: repository.owner.login,
+						repo: repository.name,
+						pull_number: pull.number
+					})).data;
+
+				//TODO: Add getGithubUser call to new GH client
```

Comment:
I believe we should do it in this PR, otherwise we'll be using both clients on Pull Reqeusts path

Reply:
Will add this call to new client